### ISSUE TITLE
Dynamic Variables suggestion list not displayed for RAW (XML, HTML, Text) format.

### ISSUE THREAD

**Issue by Pricilla09** [a85/POSTMan-Chrome-Extension#9356]
**Description**

Dynamic variables suggestion list is not provided when trying to pass the body as RAW (XML, HTML, Text), where as its listed for JSON and JS formats.

**Steps to Reproduce**

1. Launch latest postman version.
2. Create a POST request with endpoint (https://www.dataaccess.com/webservicesserver/NumberConversion.wso)
3. Select the Body as RAW XML and paste the below content:
```
<?xml version="1.0" encoding="utf-8"?>
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
  <soap:Body>
    <NumberToDollars xmlns="http://www.dataaccess.com/webservicesserver/">
      <dNum>200</dNum>
    </NumberToDollars>
  </soap:Body>
</soap:Envelope>
```
4. For attribute "**dNum**" try passing the Dynamic variables and start typing {{$random.. No suggestions listed here,
but the same is listed for JSON and JS formats.

**Screenshots or Videos**

![image](https://user-images.githubusercontent.com/71498483/101570143-c55c9980-3a10-11eb-8872-26a5180b928d.png)

![image](https://user-images.githubusercontent.com/71498483/101570220-e8874900-3a10-11eb-9f77-e0dabd1f30eb.png)

![image](https://user-images.githubusercontent.com/71498483/101570187-d86f6980-3a10-11eb-9b63-1793eeab7bea.png)

**Environment Information**

 - Native Desktop App
 - Postman Version:  v7.36.0
 - Operating System: Windows 10 Enterprise.

**Additional Context**
Sorry I tried to locate the issue if the same kind is reported already, I couldn't find. 
Is this something not possible or already in the roadmap, please close the issue.

**Comment by DannyDainton** [comment]
Hey @Pricilla09 

Would you be able to check if the Dynamic Variables show when you don't use the `$` symbol first? So just start typing `{{ran` etc.  

![image](https://user-images.githubusercontent.com/17089546/101637596-dfa07280-3a24-11eb-8d91-609e517eb5e5.png)

**Comment by Pricilla09** [comment]
@DannyDainton Thanks a lot Danny, its working now. Silly me!! But is this the expected? Also here the closing braces to be provided manually. 

![image](https://user-images.githubusercontent.com/71498483/101701896-cc3fe680-3aba-11eb-8510-92dd08260ff8.png)

![image](https://user-images.githubusercontent.com/71498483/101702028-07dab080-3abb-11eb-9b4e-71ef0d5d0816.png)

**Comment by DannyDainton** [comment]
I don't believe this is the expected behaviour and should be handled the same way we do for the JSON payloads.

I've raised this with the team and we'll update this thread once we have more information.

Thanks for flagging this up 🏆

**Comment by Pricilla09** [comment]
Great to know Danny! Thank you :-)
